src/components/tl/ucp/tl_ucp_team.c

```diff
@@ -91,11 +91,15 @@ UCC_CLASS_INIT_FUNC(ucc_tl_ucp_team_t, ucc_base_context_t *tl_context,
     }
 
     if (self->topo && !IS_SERVICE_TEAM(self) && self->topo->topo->sock_bound) {
-        max_radix       = ucc_min(UCC_TL_TEAM_SIZE(self),
-                                  ucc_topo_min_socket_size(self->topo));
-
+        max_radix = ucc_min(UCC_TL_TEAM_SIZE(self),
```

Comment:
nit: in my opinion more readable as
```suggestion
        max_radix = ucc_topo_max_ppn(self->topo) == 1 ? UCC_TL_TEAM_SIZE(self) : ucc_min(UCC_TL_TEAM_SIZE(self),ucc_topo_min_socket_size(self->topo));
```
Also, don't we have always `ucc_topo_min_socket_size(self->topo) <= UCC_TL_TEAM_SIZE(self)`?

Reply:
>> Also, don't we have always ucc_topo_min_socket_size(self->topo) <= UCC_TL_TEAM_SIZE(self)

yes, but if ucc_topo_min_socket_size() function cannot determine min_socket_size it will return UCC_RANK_MAX